tests/test_analytical_simulator.py

```diff
@@ -0,0 +1,68 @@
+#   Copyright 2022 Entropica Labs
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+import unittest
+
+from openqaoa.backends.simulators.qaoa_analytical_sim import QAOABackendAnalyticalSimulator 
+from openqaoa.utilities import random_k_regular_graph
+from openqaoa.workflows.optimizer import QAOA, RQAOA
+from openqaoa.problems.problem import QUBO, MaximumCut
+from openqaoa.devices import create_device
+
+"""
+A set of tests for FastQAOA: see the notebook Test_Examples.ipynb for explanations of how the
+expected answers are derived.
+"""
+def run_Rqaoa_experiment():  
```

Comment:
The tests are not correctly implemented: see any of the other test for the template (they need to be part of a class and they need to have names that begin with `test_`

Reply:
I have implemented the changes you suggested. Can you take a look here: https://github.com/entropicalabs/openqaoa/blob/5a51f2c8c4f2fc5deaf68fd7583ee9afefe3f7fd/tests/test_analytical_simulator.py#L53